backend/Dockerfile

```diff
@@ -1,9 +1,30 @@
-FROM openjdk:11.0.10-slim
-WORKDIR /quarkus-app/
-COPY .mvn/wrapper/ ./.mvn/wrapper/
-COPY pom.xml mvnw ./
-RUN ./mvnw clean install
+FROM maven:3.8.4-openjdk-11-slim as build
+
+RUN curl --location --output graalvm.tgz https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.3.4/graalvm-ce-java11-linux-amd64-20.3.4.tar.gz \
+    && tar -xzf graalvm.tgz \
+    && mv graalvm-* /opt/graalvm \
+    && /opt/graalvm/bin/gu install native-image \
+    && apt update \
+    && apt install build-essential libz-dev zlib1g-dev -y
```

Comment:
You could clean up the apt cache with
`rm -rf /var/lib/apt/lists/*`
This could save some space